consensus/istanbul/validator/weighted.go

```diff
@@ -681,12 +705,24 @@ func filterPoorValidators(weightedValidators []*weightedValidator, stakingAmount
 		} else {
```

Comment:
@jeongkyun-oh When this loop check the amount, it can check if the node is `govNode`. Then even if the node doesn't have enough staking, it can be skipped when the node is govNode.

This code works like demoting and promoting.
But I think demoting only normal node except govNode is better. What do you think?

Reply:
@KimKyungup  The reason is that the governing node should be demoted when no one has enough staking amount of KLAYs. Otherwise, only the governing node will be a proposer and the committee member.